src/codelabs/async-await/index.md

```diff
@@ -417,11 +432,12 @@ width="100%" >
 
```

Comment:
In ToC, it's under "Handling errors". I guess it should be move a level up?

Reply:
@kwalrath I could be wrong, but I think ToC level for exercises and examples may be a point of disagreement here. If it is, I'll leave it to you two.

Reply:
I'm out next Monday. Feel free to ignore this one if other content are ready to go. It's a minor point. We can revisit later.

Reply:
Agreed that this exercise doesn't belong under Handling errors. Changing `###` to `##` makes sense here.